### ISSUE TITLE
Question about the loss and pretrained FCN-8s

### ISSUE THREAD

**Issue by netw0rkf10w** [sadeepj/crfasrnn_keras#43]
Hi Sadeep,

Thanks for making this code available!

I would like to ask a few questions regarding the paper.

When you plugged the CRF layer to the pretrained FCN-8s and trained end-to-end, did the (softmax) **loss** actually go down? How many epochs did you train the model to see that it eventually went down? I tried training end-to-end ('lr=1e-13, momentum=0.99' as described in the paper) with the pretrained FCN-8s weights extracted from your model but the loss does not seem to go down :(

Could you please share the (Keras) pretrained FCN-8s weights that you used? Also, did you use any data augmentation?

Thank you very much in advance!
Best regards.

**Comment by chciw** [comment]
I have the same question:(
